src/Parser/DeathKnight/Frost/Modules/Features/Checklist.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import ITEMS from 'common/ITEMS';
+
+import SpellLink from 'common/SpellLink';
+import Wrapper from 'common/Wrapper';
+
+import CoreChecklist, { Rule, Requirement } from 'Parser/Core/Modules/Features/Checklist';
+import Abilities from 'Parser/Core/Modules/Abilities';
+import { PreparationRule } from 'Parser/Core/Modules/Features/Checklist/Rules';
+import { GenericCastEfficiencyRequirement } from 'Parser/Core/Modules/Features/Checklist/Requirements';
+import CastEfficiency from 'Parser/Core/Modules/CastEfficiency';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import LegendaryUpgradeChecker from 'Parser/Core/Modules/Items/LegendaryUpgradeChecker';
+import LegendaryCountChecker from 'Parser/Core/Modules/Items/LegendaryCountChecker';
+import PrePotion from 'Parser/Core/Modules/Items/PrePotion';
+import EnchantChecker from 'Parser/Core/Modules/Items/EnchantChecker';
+
+import AlwaysBeCasting from './AlwaysBeCasting';
+import RunicPowerDetails from '../RunicPower/RunicPowerDetails';
+import RuneTracker from './RuneTracker';
+
+class Checklist extends CoreChecklist {
+  static dependencies = {
+    abilities: Abilities,
+    castEfficiency: CastEfficiency,
+    combatants: Combatants,
+    legendaryCountChecker: LegendaryCountChecker,
+    legendaryUpgradeChecker: LegendaryUpgradeChecker,
+    prePotion: PrePotion,
+    alwaysBeCasting: AlwaysBeCasting,
+    enchantChecker: EnchantChecker,
+    runicPowerDetails: RunicPowerDetails,
+    runeTracker: RuneTracker,
+  };
+
+  rules = [
+    new Rule({
+      name: 'Use core spells as often as possible',
+      description: <Wrapper>Spells with short, static cooldowns like <SpellLink id={SPELLS.REMORSELESS_WINTER.id}/> and <SpellLink id={SPELLS.CHAINS_OF_ICE.id}/>(when using Cold Heart) should be used as often as possible</Wrapper>,
+      requirements: () => {
+        const combatant = this.combatants.selected;
+        return [
+          new GenericCastEfficiencyRequirement({
+            spell: SPELLS.REMORSELESS_WINTER,
+          }),
+          new GenericCastEfficiencyRequirement({
+            spell: SPELLS.CHAINS_OF_ICE,
```

Comment:
CoI isn't in abilities.js for frost (a lot of other abilities are probably missing aswell, it's quite short). I would also recommend tracking it using the cold heart module to get a requirement for this as setting CoI to having a cooldown would throw errors whenever it is used early (it does this for UH atm) and the Cold Heart module was updated to have a lot of extra rules for casting early/late iirc.

Reply:
Whoops, forgot to add CoI to abilities.  The rules for casting CH are different for frost and uh and I'm inclined to forgo adding a new module for frost.  It's lazy of me but CH is rarely used with frost and with BFA in 4 months it'll become a moot point.  I should go and make sure UHs module updates the cast efficiency correctly, though.  CH is widely used so the majority of UH players who use the tool will get those errors.